docs/dev-practices.md

```diff
@@ -2,6 +2,10 @@
 
 This page describes development practices for this codebase.
 
+## Auto-formatting
+
+[cljfmt](https://cljdoc.org/d/cljfmt/cljfmt/0.9.0/doc/readme) is a common formatter used for Clojure, analogous to Prettier for other languages. While we do not format/indent consistently with cljfmt across the whole codebase, we recommend that you do so for code that you change/add. You can do so easily with the [Calva](https://marketplace.visualstudio.com/items?itemName=betterthantomorrow.calva) extension in [VSCode](https://code.visualstudio.com/): It will (mostly) indent your code correctly as you type, and you can move your cursor to the start of the line(s) you've written and press `Tab` to auto-indent all Clojure forms nested under the one starting on the current line.
```

Comment:
> we recommend that you do so for code that you change/add

We would rather that contributions respect existing whitespace conventions of the files they modify. We'll eventually put this advice in a CONTRIBUTING.md, hopefully soon